script/preview.js

```diff
@@ -12,7 +12,7 @@ const defaultHost = HOSTNAMES[defaultBuildtype];
 const defaultContentDir = '../../../../../vagov-content/pages';
 
 const COMMAND_LINE_OPTIONS_DEFINITIONS = [
-  { name: 'buildtype', type: String, defaultValue: defaultBuildtype },
+  { name: 'buildtype', type: String, defaultValue: process.env.PREVIEW_BUILD_TYPE || defaultBuildtype },
```

Comment:
What is the different build type for?

Reply:
@jbalboni The idea is just to make it easier to configure. I just wanted to add the option to use and ENV var instead of cli flag. Does that make sense?

Reply:
Yep, it does.